client/python/tests/test_cache_map.py

```diff
@@ -0,0 +1,95 @@
+from cache_map import *
+
+import random
+
+TEST_CNT = 10000
```

Comment:
This abbreviation is likely to be pronounced cunt. Let's spell out count. Thanks.